### PR TITLE
Create dependencyci.yml

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: dependencyci.yml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 # don't run unmaintained test on any dependencies in Gemfile
```
Comment (mpirri) [inline dependencyci.yml]: Is this comment out of date now?  (since we don't specifically call out the `Gemfile` in the configuration...)

Comment (KrauseFx) [inline dependencyci.yml]: It's still a Gemfile, that references the gemspec 👍

Comment (mpirri) [inline dependencyci.yml]: Okay, sounds good!

**Review by mpirri — APPROVED** [review]
:+1:

**Pull request merged**